crates/node/src/abortable.rs

```diff
@@ -103,49 +120,142 @@ impl AbortableSpawner {
         status
     }
 
-    /// This method is responsible for actually spawning the async task into the
-    /// runtime.
-    fn spawn_abortable_task<A, F, R>(
+    /// Run all the spawned tasks to completion.
+    pub async fn run_to_completion(mut self) {
+        let pinned_task = self.pinned.take();
+
+        let batch = std::mem::take(&mut self.batch);
+        let (task_ids, task_handles): (Vec<_>, Vec<_>) =
+            batch.into_iter().unzip();
+
+        // Wait for interrupt signal or abort message
+        let aborted = self.wait_for_abort().await.child_terminated();
+
+        // Wait for all managed tasks to finish
+        match futures::future::try_join_all(task_handles).await {
+            Ok(results) => {
+                for (i, res) in results.into_iter().enumerate() {
+                    match res {
+                        Err(err) if aborted => {
+                            let who = task_ids[i];
+                            tracing::error!("{who} error: {err}");
+                        }
+                        _ => {}
+                    }
+                }
+            }
+            Err(err) => {
+                // Ignore cancellation errors
+                if !err.is_cancelled() {
+                    tracing::error!("Abortable spawner error: {err}");
```

Comment:
```suggestion
                    let who = task_ids[i];
                    tracing::error!("Abortable spawner for {who} error: {err}");
```

Reply:
this is a join error, not an error from one of the tasks